package.json

```diff
@@ -76,6 +76,12 @@
           "scope": "resource",
           "description": "Collapse parent items by default.",
           "type": "boolean"
+        },
+        "memo.syntaxDecorations.enable": {
+          "default": true,
+          "scope": "resource",
+          "description": "Enable decorations for wiki-links and other special syntax.",
```

Comment:
```suggestion
          "description": "Enable decorations for links.",
```

Reply:
So far, only link decorations are supported. Also, in Memo, those called either links or refs due to relative paths support, which makes a difference to wikilinks. I suggest omitting `wiki-` part.